src/components/Textarea/index.js

```diff
@@ -0,0 +1,48 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core'
+import React from 'react'
+
+import styles from './Textarea.styles'
+import { visuallyHidden } from '../../lib/theme/'
```

Comment:
could we import this into the styles file instead and add an `activeLabel` that uses this? similar to how izzy did `...buttonReset` from the theme?

Reply:
Great suggestion, I hadn't see her do that with `buttonReset` yet. Thanks @lizbanach!